api/tools/src/main/java/org/pmiops/workbench/tools/ServiceAccountAPIClientFactory.java

```diff
@@ -1,6 +1,6 @@
 package org.pmiops.workbench.tools;
 
-import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
 import java.util.Arrays;
 import org.pmiops.workbench.firecloud.ApiClient;
```

Comment:
Why do we need to use a factory class instead of defining a bean in a configuration somewhere? If we had an ApiUrlProvider it could be beans all the way down...